flake.nix

```diff
@@ -3,40 +3,62 @@
 
   inputs = {
     nixpkgs.url = "github:NixOS/nixpkgs/nixos-unstable";
-    flake-utils.url = "github:numtide/flake-utils";
     ixx = {
       # match version with npm package
       url = "github:NuschtOS/ixx/v0.0.6";
       inputs = {
         nixpkgs.follows = "nixpkgs";
-        flake-utils.follows = "flake-utils";
       };
     };
   };
 
-  outputs = { nixpkgs, flake-utils, ixx, ... }:
-    flake-utils.lib.eachDefaultSystem
-      (system:
+  outputs =
+    { nixpkgs, ixx, ... }:
+    let
+      inherit (nixpkgs) lib;
+      systems = [
+        "x86_64-linux"
+        "aarch64-linux"
+        "x86_64-darwin"
+        "aarch64-darwin"
```

Comment:
I am not such a big fan of mentioning darwin here explicitly since we never tested it. We could easily avoid this with a comment but in the end: before we didn't say anything ;)